<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DrawerGL" name="drawer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1200</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>690</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>420</y>
      <width>691</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="activateDrawButton">
         <property name="text">
          <string>Визуализация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Старт/Стоп</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="worldSettingsMenu">
           <property name="text">
            <string>Настройки мира</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newPopulationSettingsButton">
           <property name="text">
            <string>Создать популяцию</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveGenomsButton">
           <property name="text">
            <string>Сохранить популяцию</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadGenomsButton">
           <property name="text">
            <string>Загрузить популяцию</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item row="0" column="1">
        <widget class="GenomsListWidget" name="listGenoms">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSlider" name="speedSlider">
         <property name="value">
          <number>99</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawerGL</class>
   <extends>QOpenGLWidget</extends>
   <header>drawergl.h</header>
  </customwidget>
  <customwidget>
   <class>GenomsListWidget</class>
   <extends>QListWidget</extends>
   <header>genomslistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
